app/src/main/java/org/mozilla/fenix/components/metrics/LeanplumMetricsService.kt

```diff
@@ -101,16 +99,21 @@ class LeanplumMetricsService(
 
     @Suppress("ComplexMethod")
     override fun start() {
+        val preferencesLoaded = scope.async(Dispatchers.IO) {
```

Comment:
Scope is already defined as `Dispatchers.IO` – could we make this a part of the leanplumJob too? (see below)